prepo.py

```diff
@@ -1,25 +1,25 @@
-# -*- coding: utf-8 -*-
-#/usr/bin/python2
-'''
-By kyubyong park. kbpark.linguist@gmail.com.
-https://www.github.com/kyubyong/dc_tts
-'''
-
-from __future__ import print_function
-
-from utils import load_spectrograms
-import os
-from data_load import load_data
-import numpy as np
-import tqdm
-
-# Load data
-fpaths, _, _ = load_data() # list
-
-for fpath in tqdm.tqdm(fpaths):
-    fname, mel, mag = load_spectrograms(fpath)
-    if not os.path.exists("mels"): os.mkdir("mels")
-    if not os.path.exists("mags"): os.mkdir("mags")
-
-    np.save("mels/{}".format(fname.replace("wav", "npy")), mel)
+# -*- coding: utf-8 -*-
+#/usr/bin/python2
+'''
+By kyubyong park. kbpark.linguist@gmail.com.
+https://www.github.com/kyubyong/dc_tts
+'''
+
+from __future__ import print_function
+
+from utils import load_spectrograms
+import os
+from data_load import load_data
+import numpy as np
+import tqdm
+
+# Load data
+fpaths, _, _ = load_data() # list
+
+for fpath in tqdm.tqdm(fpaths):
+    fname, mel, mag = load_spectrograms(fpath)
+    if not os.path.exists("mels"): os.mkdir("mels")
+    if not os.path.exists("mags"): os.mkdir("mags")
+
+    np.save("mels/{}".format(fname.replace("wav", "npy")), mel)
     np.save("mags/{}".format(fname.replace("wav", "npy")), mag)
```

Comment:
is there a reason the last two lines are double? It isn't like that in the original code. Maybe just delete it

Reply:
It actually is like that in the original code. They are not exactly the same. One contains "mel" and the other one contains "mag".

Reply:
Oh right, that was our mistake! Should look more closely ;-)